src/reanimated2/hook/useAnimatedGestureHandler.ts

```diff
@@ -18,38 +16,43 @@ export interface GestureHandlers<T, TContext extends Context> {
   onFinish?: Handler<T, TContext>;
 }
 
-export enum EventType {
-  UNDETERMINED = 0,
-  FAILED,
-  BEGAN,
-  CANCELLED,
-  ACTIVE,
-  END,
-}
+export const EventType = {
```

Comment:
Why do we change enum to object?

Reply:
cc @j-piasecki on this one, we had a conversation about it a long while ago and from what I remember we concluded that using an object is better here, but I don't remember exact reasons of this. Change was made in #3772.

Reply:
IIRC, it's related to change below, I don't think `(typeof EventType)[keyof typeof EventType]` would work on enum. Could be wrong though.